textproc/asciidoctor/Portfile

```diff
@@ -5,7 +5,13 @@ PortGroup           ruby 1.0
 
 ruby.setup          asciidoctor 2.0.18 gem {} rubygems ruby31
 name                asciidoctor
-revision            1
+revision            2
```

Comment:
no need to increase the "revision" as you are fixing a build issue. Everything that did build doesn't change..